pvr.argustv/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.argustv"
-  version="20.3.0"
+  version="20.3.1"
```

Comment:
Sorry, I spoke too soon. As you are adding functionality the version number should 2.4.0. You would use 2.3.1 if you had fixed a bug.

Reply:
20., not 2. ;-)

Reply:
Whoops.

Reply:
@phunkyfish 
I thought of this is a bug fix. It seemed to be a misconfiguration.  I was torn though about it being added functionality vs a bug fix though.  I can change it.  Are you sure you wouldn't consider it new functionality?

Reply:
It's new functionality. There is no misbehavior that your PR fixes. This, please update the add-on minor version => 20.4.0